apps/docs/src/examples/extras/positional-audio/Disc.svelte

```diff
@@ -15,10 +16,11 @@
       autoStart: false
     }
   )
-  $: {
+
+  $effect(() => {
```

Comment:
i think we can write this $effect as

```typescript
$effect(() => {
  if (discSpeed > 0) {
    start()
    return () => {
      stop()
    }
})
```


then we don't have to reference `$started`

so the task is started if discSpeed > 0. and it's stopped before the effect reruns. but it never needs to be stopped if it was never started.

could go even further and break it out into a derived to prevent additional work.

```typescript
const discSpeedGreaterThanZero = $derived(discSpeed > 0)

$effect(() => {
  if (discSpeedGreaterThanZero) {
    // ...
  }
})
```

if `discSpeed` goes from say 1 to 2 then `discSpeedGreaterThanZero` goes from `true` to `true` and the effect cleanup won't run because its dependencies haven't changed.